### PR TITLE
Uploading Assignment8 and made changes in Assignment 7

### PR DESCRIPTION
**Pull request by monicapathania**

### CODE REVIEW

File: Monica/html/Day7Assingment/style.css [hunk 1]
```diff
@@ -167,4 +167,5 @@ header{
     .con1{
         margin: 150px;
         margin-top: 20px;
-    }
\ No newline at end of file
+    }
+
```
Comment (omerana14) [inline Monica/html/Day7Assingment/style.css]: Add blank space at the end of the program

File: Monica/html/Day8assignment/Assignment8.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                placeat rem velit quibusdam harum non iste necessitatibus ducimus eum optio. Explicabo, nemo illum.
+                Molestias voluptatem sequi libero non cum vero eligendi labore, earum quo veritatis iusto obcaecati
+                laudantium
+                quos sit tempora! Quas dolorum adipisci ab quis voluptates nemo nisi ut cum consequatur porro!
+            </p>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (omerana14) [inline Monica/html/Day8assignment/Assignment8.html]: same here

File: Monica/html/Day8assignment/style.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.clss_c > p {
+  float: right;
+}
+
+.clss_c > p > img {
+  float: left;
+  margin-right: 10px;
+}
```
Comment (omerana14) [inline Monica/html/Day8assignment/style.css]: same here

File: Monica/html/Day8assignment/Assignment8.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```
Comment (MKadeer17) [inline Monica/html/Day8assignment/Assignment8.html:4]: little bit indentation only

**Review by MKadeer17 — APPROVED** [review]
Only indentation before the head and body tag

**Review by Pawan1005 — APPROVED** [review]

File: Monica/html/Day8assignment/Assignment8.html (hunk 4 above)
Comment (monicapathania) [reply Monica/html/Day8assignment/Assignment8.html:4]: okay

**Review by omerana14 — APPROVED** [review]

**Pull request merged**